fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/SideRailTokens.kt

```diff
@@ -38,4 +38,8 @@ open class SideRailTokens : IControlToken, Parcelable {
     open fun headerPadding(sideRailInfo: SideRailInfo): PaddingValues {
         return PaddingValues(16.dp)
     }
+    @Composable
+    open fun backgroundColor(sideRailInfo: SideRailInfo): Color {
```

Comment:
return brush instead of Color & rename function to backgroundBrush

Reply:
made the changes